src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```diff
@@ -3,7 +3,7 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>net6.0</TargetFramework>
     <AssemblyName>func</AssemblyName>
```

Comment:
Please update this script so that npm picks up the win-arm64 bits as well: https://github.com/Azure/azure-functions-core-tools/blob/v4.x/src/Azure.Functions.Cli/npm/lib/install.js#L27

Or file an issue and assign it to me if you don't want to deal with javascript :/